astroquery/gemini/core.py

```diff
@@ -390,6 +417,18 @@ def query_raw(self, *args, **kwargs):
         js = response.json()
         return _gemini_json_to_table(js)
 
+    def get_file(self, filename, *, timeout=None):
```

Comment:
oh, this should be covered with a test though.

Reply:
ok I added a test, added an optional directory parameter to make it easier, and updated the docs